app/controllers/api/base_controller.rb

```diff
@@ -0,0 +1,50 @@
+module Api
+  class BaseController < ApplicationController
+    protect_from_forgery with: :null_session
+
+    skip_before_action :set_current_team
+    skip_before_action :run_first_time_setup
+    skip_before_action :ensure_team_exists_for_subdomain!
+    skip_before_action :require_login
+    skip_before_action :user_belongs_to_team!
+
+    before_action :require_api_token
+    before_action :set_current_team
+
+    private
+
+    def render_errors(object)
+      render json: {message: "Validation Failed", errors: object.errors.full_messages}, status: 422
```

Comment:
Good call, I usually use full_messages cause I don’t like how it returns otherwise like:

```rb
[
  [:name, ["can’t be blank"]]
]
```

Have reworked it a bit so it returns like:

```rb
[
 { field: "name", errors: ["can’t be blank"] }
]
```